lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless government_data_availible?
+
+      return GovukHealthcheck::WARNING if data_stale?
+
+      GovukHealthcheck::OK
+    end
+
+    def details
+      {
+        critical: "No government data availible",
+        warning: warning_details_content,
+      }
+    end
+
+    def enabled?
+      true
+    end
+
+  private
+
+    def government_data_availible?
+      BulkData::GovernmentRepository.new.cache_populated?
+    end
+
+    def data_stale?
+      repository = BulkData::GovernmentRepository.new
+      repository.age_of_cache > 6.hours
+    end
+
+    def warning_details_content
+      data_age = BulkData::GovernmentRepository.new.age_of_cache
+      "Government data not refreshed in #{data_age / 3600} hours #{data_age / 60} minutes"
```

Comment:
This doesn't look correct - and also seems to be missing a test - as if you had 7200 as the age you'd get an output of "Government data not refreshed in 2 hours 120 minutes". You'll probably want to pull out the modulus operator.

Also using ActiveSupport::Duration and division for conversion makes some pretty strange intermediary numbers for example:
```
irb(main):034:0> 7200.seconds / 3600
=> 2 seconds
```

So might be actually be a not very friendly class to use and better to use an integer.

Reply:
Rookie error on this one, have adjusted the output and written a test for it. I've left the `cache_age` method returning an `ActiveSupport::Duration` and turned that into an integer on the `warning_details_content` method. It means the return clauses above look a bit nicer as I can check for `@government_repo.cache_age > 6.hours` rather than `@government_repo.cache_age > 21600`